src/actions/Admin/index.js

```diff
@@ -104,27 +104,28 @@ const methods = {
             if(!error && response.statusCode === 200 && body.data && body.data.streams) {
                 const action = 'saved';
                 const streams = body.data.streams;
-                self.dispatch(constants.ADMIN.LOAD_TWITTER_ACCTS, {streams, action});
+                self.dispatch(constants.ADMIN.LOAD_TWITTER_ACCOUNTS, {streams, action});
             }else{
                 console.error(`[${error}] occured while processing message request`);
             }
        });
     },
 
-    load_twitter_accts (siteId) {
-        let self = this;
-        AdminServices.getTwitterAccounts(siteId, (error, response, body) => {
-                    if (!error && response.statusCode === 200) {
-                        const streams = body.data.streams;
-                        let action = false;
-                        self.dispatch(constants.ADMIN.LOAD_TWITTER_ACCTS, {streams, action});
-                    }else{
-                        let error = 'Error, could not load twitter accts for admin page';
-                        self.dispatch(constants.ADMIN.LOAD_FAIL, { error });
-                    }
-        });
+    load_twitter_accounts () {
```

Comment:
Nit: might as well fix the space before the parens if we're already touching this line :)